docs/src/prototypes/chatMessageWithPopover/ChatMessageWithPopover.tsx

```diff
@@ -0,0 +1,102 @@
+import { Chat, Provider } from '@stardust-ui/react'
+
+import * as React from 'react'
+import Popover from './Popover'
+
+const janeAvatar = {
+  image: 'public/images/avatar/small/ade.jpg',
+  status: { color: 'green', icon: 'check' },
+}
+
+class ChatMessageWithPopover extends React.Component {
```

Comment:
Split this into two files - `ChatMessageWithPopover` and `ChatWithPopover`. It's confusing now that the name of the file is ChatMessageWithPopuover and we are exporting as default ChatWithPopover. Let's keep consistency for having one class per file.